bootstrap.d/app-editors.y4.yml

```diff
@@ -65,6 +65,37 @@ packages:
       - args: ['cp', '@SOURCE_ROOT@/extrafiles/nanorc', '@THIS_COLLECT_DIR@/home/managarm/.nanorc']
       - args: ['cp', '@SOURCE_ROOT@/extrafiles/root-nanorc', '@THIS_COLLECT_DIR@/root/.nanorc']
 
+  - name: openvi
+    architecture: '@OPTION:arch@'
+    metadata:
+      summary: Portable OpenBSD vi for UNIX systems
+      description: OpenVi is an enhanced and portable implementation of the Berkeley vi / ex text editor
+      spdx: 'BSD-3-Clause'
+      website: 'https://github.com/johnsonjh/OpenVi'
+      maintainer: "Jeffrey H. Johnson <trnsz@pobox.com>"
+      categories: ['app-editors']
+    source:
+      subdir: 'ports'
+      git: 'https://github.com/johnsonjh/OpenVi.git'
+      tag: '7.6.31'
+    tools_required:
+      - system-gcc
+    pkgs_required:
+      - mlibc
+      - ncurses
+    build:
```

Comment:
Missing revision tag, it should be set to 1 for new ports. See the other ports in the file for how it should look. The vim port is a good candidate to look at.